### PR TITLE
installation with curl and bash

### PR DESCRIPTION
**Pull request by spacewander**
Does it work well for #379 ?

### CODE REVIEW

File: Installation.md [hunk 1]
```diff
@@ -24,5 +24,5 @@ $ [sudo] make install
 One-liner:
 
 ```bash
-$ (cd /tmp && rm -rf git-extras && echo "Setting up 'git-extras'...." && git clone https://github.com/tj/git-extras.git &> /dev/null && cd git-extras && git checkout $(git describe --tags $(git rev-list --tags --max-count=1)) &> /dev/null && sudo make install)
+curl -s https://raw.githubusercontent.com/tj/git-extras/master/install.sh | [sudo] bash /dev/stdin
 ```
```
Comment (hemanth) [inline Installation.md]: That URL is giving a 404, anyway I have shortened the same URL to `http://git.io/git-extras-setup` so if you fix that URL, we can use the shortened URL ;) :+1:

Comment (hemanth) [inline Installation.md]: Still 404!

Comment (hemanth) [inline Installation.md]: Well, it took some time to update.

Final version: `curl -sSL http://git.io/git-extras-setup | sudo bash /dev/stdin`  :)

**Comment by apjanke** [comment]
Looks like this has an issue with the `Makefile`: in some cases, `Makefile` will prompt for user input, like if a git alias conflicts with one of the `git-extras` commands. It can't interact with the user in the `curl ... | bash` form, since `stdin` holds the script itself.

The `sh -c` form might work better, because then stdin is still available for user input.

```
bash -c "$(curl -sSL http://git.io/git-extras-setup)"
```

**Pull request merged**